cmd/tsbs_generate_queries/main.go

```diff
@@ -51,6 +53,24 @@ var useCaseMatrix = map[string]map[string]utils.QueryFillerMaker{
 		iot.LabelDailyActivity:                 iot.NewDailyTruckActivity,
 		iot.LabelBreakdownFrequency:            iot.NewTruckBreakdownFrequency,
 	},
+	"finance": {
+		finance.LabelLastPrice:                     finance.NewLastPrice,
+		finance.LabelOpenHighLowClose + "-5m":      finance.NewOpenHighLowClose(5 * time.Minute),
+		finance.LabelOpenHighLowClose + "-1h":      finance.NewOpenHighLowClose(1 * time.Hour),
+		finance.LabelOpenHighLowClose + "-24h":     finance.NewOpenHighLowClose(24 * time.Hour),
```

Comment:
The query is hard coded with time unit "minute", does it work to pass in an arbitrary time value here? Similar questions for other queries in the list.

Reply:
Yep, it will be converted to the equivalent in minutes.

Reply:
Ok I see, so it will always be converted into minutes representation.